public/js/visual-diff.js

```diff
@@ -0,0 +1,4 @@
+$('iframe').load( function() {
```

Comment:
Is this a handler on all iframes everywhere all the time? That is bound to cause issues if so and needs to be scoped with a data attribute or something.